commit f9431baafd69372698cd5ed87d377633064dafa1 (refs/changes/37/2337/2)
Author: Joshua M. Clulow <jmc@joyent.com>
Date:   2017-08-06T00:11:50+00:00 (2 years, 2 months ago)
    
    OS-5671 installer should not set "erase" tty option
    Reviewed by: Jason King <jason.king@joyent.com>
    Approved by: Jason King <jason.king@joyent.com>

diff --git a/overlay/generic/smartdc/lib/smartos_prompt_config.sh b/overlay/generic/smartdc/lib/smartos_prompt_config.sh
index 0b487931..6a0b37ac 100644
--- a/overlay/generic/smartdc/lib/smartos_prompt_config.sh
+++ b/overlay/generic/smartdc/lib/smartos_prompt_config.sh
@@ -1019,7 +1019,6 @@ done
 updatenicstates
 
 export TERM=xterm-color
-stty erase ^H
 
 trap sig_doshell SIGINT
 
